noodles-editor/src/noodles/components/project-not-found-dialog.tsx

```diff
@@ -2,7 +2,7 @@ import * as Dialog from '@radix-ui/react-dialog'
 import { Cross2Icon } from '@radix-ui/react-icons'
 import cx from 'classnames'
 import { useCallback, useState } from 'react'
-import newProjectJSON from '../../../public/noodles/new.json'
+import newProjectJSON from '../../../public/noodles/new/noodles.json'
```

Comment:
Vite doesn't like that we're importing from the public folder. This should probably be a `fetch`

Reply:
#56